babybuddy/static_src/scss/global.scss

```diff
@@ -52,3 +52,8 @@
 .icon-2x {
     font-size: 1.65em;
 }
+
+// All modals
+.modal-content {
+    color: theme-color('dark');
+}
```

Comment:
Without this, the modal was still rendered in the "light" theme and was barely readable.

Reply:
Interesting. I'll take a look at this when I do some testing. May want to customize further.